data/faq.md

```diff
@@ -31,40 +53,40 @@ A. To get an idea of what to expect, please have a look at the
 [EP2021 programme](https://ep2021.europython.eu/events/sessions/) and the
 [EP2019 programme](https://ep2019.europython.eu/events/sessions/).
 
-To select talks, we run a Call for Proposals (CFP). Talks are then curated based
-on the input from the community voting and expert panel reviews. Make sure to subscribe to our
-[EuroPython Blog](https://blog.europython.eu/#/portal/signup) for updates.
+The conference will feature around 20 tutorials/workshops, 120 talk sessions and other interactive sessions such as Posters, Helpdesks, Panel Discussions and Maker Space.
 
-We aim to have a very diverse set of talks, tutorials, workshops, posters and
-keynotes, covering many different angles of where and how Python can be used to
-solve problems or do interesting things.
+We aim to have a very diverse programme, covering many different angles of where and how Python can be used to solve problems or do interesting things.
 
 ## **Q. When do you accept talk proposals?**
 
-A. The [Call for Proposals](https://program.europython.eu/europython-2022/cfp) (CFP) opened on 8 March.
+A. The [Call for Proposals](https://program.europython.eu/europython-2022/cfp) (CFP) ran between 8 March and 3 April.
+
+We are thrilled to have received a record high of 429 proposals on a diverse range of topics.
+
+## **Q. How do you review and select talks?**
+
+A. We believe that a good conference consists of popular talks as well as new or niche topics - all by and for our diverse community. Thus after Call for Proposals (CFP) finishes, submissions go through two round of unbiased reviews: speakers' names will not be displayed.
+
+- community voting: open to all 2015-2022 EuroPython conference ticket holders + 2022 talk proposal submitters. Submissions are in random order for each voter. Please check out our [talk voting page](https://ep2022.europython.eu/voting) for details.
+
+- panel feedback: we invite a diverse group of experienced reviewers to refine & improve on submissions within their areas of expertise. The feedback helps the programme team make informed decisions while evaluating the quality of each submission. [Submit your interest here](https://t.co/JyOdGb71f8)
```

Comment:
No need for "we invite". Why not: "a diverse groups of experienced reviewers will refine and give feedback on the submissions within their realm of expertise." There's no guarantee they'll improve the submissions..!

"Their insights help the programme team refine and plan a programme that truly reflects, informs and stimulates our EuroPython family."